spec/views/pages/conditions/routing_page.html.erb_spec.rb

```diff
@@ -22,13 +22,41 @@
     expect(rendered).to have_css("p.govuk-body", text: t("routing_page.body_text"))
   end
 
-  it "contains a fieldset legend asking a user to select a question page" do
-    expect(rendered).to have_css(".govuk-fieldset__legend", text: t("routing_page.legend_text"))
-    expect(rendered).to have_css("div.govuk-hint", text: t("routing_page.legend_hint_text"))
+  context "with fewer than 10 options" do
```

Comment:
Might be worth adding a view spec when there are no options. Do you expect to see a "blank" radio or select list. Or should we be explaining to the user why theres no valid option to select from?

Reply:
Ah yes the policy will stop that.